haskell/test/Test/Ligo/BaseDAO/Common.hs

```diff
@@ -50,7 +51,21 @@ import Ligo.BaseDAO.Types
 import Test.Ligo.BaseDAO.Common.StorageHelper as StorageHelper
 import Test.Ligo.BaseDAO.Proposal.Config (ConfigDesc, fillConfig)
 
-type OriginateFn m = m ((Address, Address), (Address, Address), TAddress Parameter, TAddress FA2.Parameter, Address)
+-- type m DaoOriginateData = m ((Address, Address), (Address, Address), TAddress Parameter, TAddress FA2.Parameter, Address)
+type OriginateFn m = m DaoOriginateData
+
+  -- pure ((dodOwner1, operator1), (dodOwner2, operator2), dodDao, dodTokenContract, admin)
```

Comment:
I think this is much better, let's remove these comments.

Reply:
Leftover, I've removed this now